tests/js/client/shell/multi/shell-validation.js

```diff
@@ -330,15 +330,37 @@ function ValidationBasicsSuite() {
     testLevelNone: () => {
       validatorJson.level = "none";
       testCollection.properties({"schema": validatorJson});
-      waitInClusterUntil(() => testCollection.properties().schema.level === validatorJson.level);
+      waitInClusterUntil(() => {
+        const shardList = testCollection.shards(true);
+        for (const [shard, servers] of Object.entries(shardList)) {
+          const endpoint = getEndpointById(servers[0]);
+          const resp = request.get(`${endpoint}/_api/collection/${shard}/properties`);
+          assertEqual(resp.statusCode, 200);
+          if (resp.json.schema.level !== validatorJson.level) {
+            return false;
+          }
+        }
+        return true;
+      });
```

Comment:
Is this function identical across all uses below? then I'd prefer making a function to call and not duplicate code.

Reply:
It is not exactly identical. There are a few different predicates used inside this function.